data/crac-io/crac-io-commons/src/main/java/com/powsybl/openrao/data/cracio/commons/api/StandardElementaryCreationContext.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.openrao.data.cracio.commons.api;
+
+import java.util.Optional;
+
+/**
+ * @author Thomas Bouquet {@literal <thomas.bouquet at rte-france.com>}
+ */
+public class StandardElementaryCreationContext implements ElementaryCreationContext {
+    protected String nativeObjectId;
+    protected String nativeObjectName;
+    protected String createdObjectId;
+    protected ImportStatus importStatus;
+    protected String importStatusDetail;
+    protected boolean isAltered;
+
+    @Override
+    public String getNativeObjectId() {
+        return nativeObjectId;
+    }
+
+    @Override
+    public Optional<String> getNativeObjectName() {
```

Comment:
I think we can avoid the hassle of handling thisOptional field by making it equal to the ID by default